### PR TITLE
Sh heading

### PR DESCRIPTION
**Pull request by sarahholder**
Created about page including heading and image with ipsom for the about paragraphs

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -4,9 +4,25 @@
...
+        <img class="spanImg" src="./bandpic.png" alt="picture of the band">
+        <h1 class="aboutHeading">About Scuurvy and Band History</h1>
+    </div>
+    <div class="aboutP">
+        <p>Prow scuttle parrel provost Sail ho shrouds spirits boom mizzenmast yardarm. Pinnace holystone mizzenmast quarter crow's nest nipperkin grog yardarm hempen halter furl. Swab barque interloper chantey doubloon starboard grog black jack gangway rutters.
+        </p>
+        <p>Deadlights jack lad schooner scallywag dance the hempen jig carouser broadside cable strike colors. Bring a spring upon her cable holystone blow the man down spanker Shiver me timbers to go on account lookout wherry doubloon chase. Belay yo-ho-ho keelhaul squiffy black spot yardarm spyglass sheet transom heave to.
+        </p>
+    </div>
+
```
Comment (lstar18) [inline about.html]: spacing :)

File: about.html [hunk 2]
```diff
@@ -4,9 +4,25 @@
...
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+    <link rel="stylesheet" href="./main.css">
     <title>Document</title>
 </head>
-<body>
-    
+<body class="aboutBody">
+    <div class="img-fluid">
+        <img class="spanImg" src="./bandpic.png" alt="picture of the band">
+        <h1 class="aboutHeading">About Scuurvy and Band History</h1>
```
Comment (lstar18) [inline about.html]: add this for the h1 so they are all uniform   <h1 class="d-flex justify-content-center">About Scuurvy</h1>

Comment (lstar18) [reply about.html]: `<h1 class="d-flex justify-content-center">About</h1>`

**Comment by lstar18** [comment]
Looks great! 👍

File: about.html (hunk 2 above)
Comment (sarahholder) [reply about.html]: What do you mean add "this"

Comment (sarahholder) [reply about.html]: want me to create a h1 css for all other pages?

**Comment by lstar18** [comment]
Looks good 👍

**Pull request merged**